src/definitions/behaviors/form.js

```diff
@@ -382,6 +450,19 @@ $.fn.form = function(parameters) {
                 }, settings.delay);
               }
             }
+          },
+          beforeUnload: function(event) {
+            if (module.is.dirty() && !submitting) {
+              var event = event || window.event;
+
+              // For modern browsers
+              if (event) {
+                event.returnValue = settings.text.leavingMessage;
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event the text itself is only supported on Internet explorer and is otherwise just a trigger (content does not matter) to display the native browser text, so keep this in mind, when writing the docs. 
Try https://jsfiddle.net/cbojyq1x/2/ . Chrome for example will always tell  you its own native leaving message instead of the value of `leavingMessage`

Reply:
Yes, I saw that too, and users will be noted about this. If we decide to not support IE in the future, i'll drop it asap 😈 !